example-client/README.md

```diff
@@ -3,11 +3,219 @@ Example CoAP client
 
 This is an example coap client application, that can be use with any LwM2M Server.
 
+# Running
+
 Usage
 -----
 
     ./run.sh <registration-url> [<keystore-file>]
 
+note: parameters are parsed like a [www URL query string](https://en.wikipedia.org/wiki/Query_string) where ? and & are delimeters for the endpoint name and account ID
```

Comment:
note -> styling -> add bolding with two asterisks around the word, and start the sentence with a capital letter. -> **Note**: Parameters....